crawler4j/checkstyle.xml

```diff
@@ -38,6 +37,7 @@
         <module name="ImportOrder">
             <property name="groups" value="java,javax,org,com,*"/>
             <property name="ordered" value="true"/>
+            <property name="separated" value="true"/>
```

Comment:
I'm not sure if it's a bug in v8.12, but without separated specified, Checkstyle treats it as separated=false, and there are about 25 source files that are then non-compliant because of blank lines between the import statements. So, specifying separated=true seemed like the path of least resistance as there are then only about 10 non-compliant files.

Aside from that, I don't really have any opinion one way or the other, so please confirm again if this is what you want to do. Cheers!

P.S. After checking the Checkstyle docs it seems more like a bug in v7.1, the default value is false if not specified, but v7.1 was not enforcing it.